Tianqi fluent

TENQEI opened the pull request.

Commit:
Physics for unsteady simulation

Commit:
Corrected spelling mistakes

teymourj commented:
@TENQEI 

Unfortunately your input for the unsteady simulation is not complete and does not match any of the project criteria. There is only one paragraph added to the end of  the Simulation file for the steady case and that paragraph is not including any technical contents.

Here is suggestions for finalizing this section: 
1- Sync the updated version of the Simulation file that was edited for the steady case study. Read through it and see the changes that was applied to. Try to understand the changes and why they are there.
2- Try edit that content and start developing on the newly constructed Physics file for the unsteady case study:
2-1 For the sections that we argued that the case is steady state and it’s corresponding term goes to zero, you should modify the content and present the corresponding arguments.
2-2 As we discussed earlier you should also discuss the physics behind boundary layer separation from the cylinder and von-karman street forming in the wake of the cylinder. This part can be added to the first paragraph (part) of the Physics file where flow is visualized for different Reynolds number.

In the end I wanted to highlight the fact that after some major changing in the structure of files on Github, now the Physics file is located in the root repository in the directory called “Common Files”. It is no longer in the FLUENT directory. Please have this in mind. When you sync the repository this changes should happen on your local machine automatically. Just make sure to edit the correct file.

Commit:
Merge branch 'master' into tianqi-fluent

Commit:
Edited physics

I added a few more words in conservation of mass and momentum but not
in dimensional analysis. I also changed some words in the very end of
the article

teymourj committed:
Finalized Physiscs file content for SFO. Ready to merge.

teymourj commented:
@TENQEI Great modifications and additions. We specifically find your inputs and point of view on evaluating the unsteady term in governing equations interesting. Your submission was revised one more time at few spots within the text body and it will be pushed to the master branch.
